Form/Type/DatePickerType.php

```diff
@@ -31,15 +31,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 )
             )
             ->add(
-                'date_example2',
-                DateType::class,
-                array(
-                    'data' => $currentDate,
-                    'widget' => 'text',
-                    'required' => false,
-                )
-            )
-            ->add(
                 'date_example3',
```

Comment:
Isn't it strange to remove example2 and not rename the other ones so that there isn't an example missing?
Now we will only have example1, example3, etc. We skipped one.

Reply:
fixed bbd97b5